heron/scheduler-core/src/java/com/twitter/heron/scheduler/ScalableScheduler.java

```diff
@@ -0,0 +1,19 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.twitter.heron.scheduler;
+
+public interface ScalableScheduler {
+  void addContainers(Integer count);
```

Comment:
This API would need additional information for heterogeneous schedulers. the YARN scheduler for eg, needs to know what resources to request for the new container. This information will come from the updated packing plan.